Lib/fontTools/ufoLib/__init__.py

```diff
@@ -2042,17 +2042,17 @@ def convertFontInfoValueForAttributeFromVersion1ToVersion2(attr, value):
 		if attr == "fontStyle":
 			v = _fontStyle1To2.get(value)
 			if v is None:
-				raise UFOLibError("Cannot convert value (%s) for attribute %s." % (repr(value), attr))
+				raise UFOLibError(f"Cannot convert value ({repr(value)}) for attribute {attr}.")
```

Comment:
you could use `{value!r}` with `!r` conversion flag, which is the same as calling `repr(value)`